### ISSUE TITLE
Auto publish to social bots for Facebook and Twitter

### ISSUE THREAD

**Issue by d4mation** [realbig/RBM_TH_LOVEFROMLOUIE#5]

**Comment by d4mation** [comment]
@BigActual Are there any updates on this? I know you said you wanted to touch base with @brashrebel about it first since he had originally brought it up with the client.

Here's what I had previously put in Slack:

> Looks like for Social Media auto-posting, the best option actually may be JetPack. They will need a WordPress**.com** account for it, but other than that it has a pretty robust system in place for doing that. https://jetpack.com/support/publicize/
>
> Otherwise, there are a couple other options. But they would need to buy a Pro version of the plugin to share both both Facebook _and_ Twitter.
> 
> https://wordpress.org/plugins/social-networks-auto-poster-facebook-twitter-g/
> https://wordpress.org/plugins/social-media-auto-publish/

> https://wordpress.org/plugins/social-media-auto-publish/ appears to support OpenGraph looking at the changelog. It was the one out of the two that I was learning more toward anyway.
> 
> The Pro version supports OpenGraph and posting to multiple Social Media accounts, among other things we may or may not use.
> 
> https://xyzscripts.com/wordpress-plugins/social-media-auto-publish/compare

**Comment by brashrebel** [comment]
We've used Post Promoter Pro on other client sites with success. It is great and made by a friend. I highly recommend. Wasentha is using it I think on PDS.

**Comment by BigActual** [comment]
Yes, too funny. I just hit send on an email about this to Loren Rex over at Generations Financial Planning. He's buying a license now. I was going to check out the system with you to see how it performs, then do this one. If Wasentha has it, give it a quick once over and if it does what we need then I'll have the client buy a license.

**Comment by d4mation** [comment]
Looks like it should do what we need. 

Looks like for Twitter it will allow you to schedule it for certain times even if having it publish immediately isn't desired.

We'll just need a License Key and for the client to authorize the plugin to publish like that in the settings screen for each Social Media Platform (Depending on how their AUTH Tokens work, we may want to wait to do this until it goes live. Otherwise they would need to be re-AUTH'd if it checks for a certain Domain making the request).

![image](https://cloud.githubusercontent.com/assets/7770631/25290614/c9e14cac-269c-11e7-8c48-e5a2ff4be333.png)
